pendulum_control/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 2.8.3)
+
+project(pendulum_control)
+find_package(ament_cmake REQUIRED)
```

Comment:
If you move the `find_package()` call below the platform check it will exit a bit fast on non Linux system.